### DISCUSSION TITLE
MONAI Model Zoo vs hugging face

### DISCUSSION THREAD

**Discussion by man-chi** [Project-MONAI/MONAI#7975] [Q&A]
some of our models are private and internal, does Model zoo provide any private repo for AI model or use Hugging Face if a private repo is required? please advise.

**Comment by KumoLiu** [comment]
Hi @man-chi,

I think so, at the moment we only have publicly accessible storage that you can use under different licenses in your bundle. But if you want to put it in a private repo, I don't think we have that option at the moment.

Correct me if l'm wrong. @yiheng-wang-nv
